plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestCachingDirectoryLister.java

```diff
@@ -60,6 +70,44 @@ protected QueryRunner createQueryRunner()
                 .build();
     }
 
+    private Table getTable(String schemaName, String tableName)
+    {
+        return fileHiveMetastore.getTable(schemaName, tableName)
+                .orElseThrow(() -> new NoSuchElementException(format("The table %s.%s could not be found", schemaName, tableName)));
+    }
+
+    private void sleep(long milliseconds) {
+        try {
+            Thread.sleep(milliseconds);
+        } catch (InterruptedException e) {
+            fail();
+        }
+    }
+
+    @Test(timeOut = 500)
+    public void testList_shouldNotTimeout() throws IOException {
```

Comment:
Great code, just add a comment on what it does. mention the default config of 1 minute timeout.